engine/consensus/matching/engine.go

```diff
@@ -511,11 +511,6 @@ func (e *Engine) checkingSealing() error {
 		Str("next_unsealed_results", nextUnsealeds.String()).
 		Logger()
 
-	if len(sealableResults) == 0 {
-		lg.Info().Msg("checking sealing finished. No results can be sealed yet")
-		return nil
```

Comment:
We should not exit earlier, because we still need to request a missing parent receipt

Reply:
💡 thanks for catching that.